app/retail/templates/emails/grants/subscription_terminated.html

```diff
@@ -60,93 +64,117 @@
     color: #5B3AF0;
-}
-.progressBar {
+  }
+
+  .progressBar {
     height: 15px;
     background-color: #D8D8D8;
     border-radius: 2.7px;
-}
-.progressCompleted {
+  }
+
+  .progressCompleted {
     height: 100%;
     background-color: #0FCE7C;
     border-radius: 2.7px;
     float: left;
-}
-.progressRemoved {
+  }
+
+  .progressRemoved {
     height: 100%;
     background-color: #CF0E0E;
     border-top-right-radius: 2.7px;
     border-bottom-right-radius: 2.7px;
     float: left;
-}
-.funding {
+  }
+
+  .funding {
     float: right;
     text-align: right;
-}
-.funding > h4 {
+  }
+
+  .funding>h4 {
     line-height: 0px;
     text-align: right;
     color: #0D0764;
     padding-top: -20px;
-}
-.progressBarLine {
+  }
+
+  .progressBarLine {
     width: 1px;
     height: 45px;
     background-color: #979797;
     float: left;
     transform: translateY(-20%);
     margin-right: 20px;
-}
-.funding > p {
+  }
+
+  .funding>p {
     line-height: 0px;
     color: #BBBDBF;
-}
+  }
+
 </style>
 <img id="newSupporter" src="{% static "v2/images/subscription-terminated.png" %}" alt="Subscription Terminated." title="Subscription Terminated">
-<h1 style="text-transform: none;">{% trans "Grant" %} '{{grant_name}}' {% trans "has been cancelled" %}.</h1> <!-- Name of grant -->
+<h1 style="text-transform: none;">{% trans "Grant" %} '{{ grant_name }}' {% trans "has been cancelled" %}.</h1>
+<!-- Name of grant -->
 <p>{%trans "Your subscription for the project has been cancelled and you will no longer be charged." %}</p>
 <br>
 
 <hr>
 <div class="grantInfo">
-        <a href="https://github.com/{{github_username}}"><img src="{{github_username}}" /></a> <!-- width="50" height="50"Input project image here -->
-        <br>
-        <a class="grantInfoName" href="https://github.com/{{github_username}}">{{github_username}}</a> <!-- Input project name here -->
-        <p>{{grant_description}}</p> <!-- Input project description here -->
-        <div class="progress">
-            <h3 style="text-transform: none;">{{funding_status}}</h3>
-            <div class="progressBar">
-                <div class="progressCompleted" style="width: {{progress_completed}}%;"> <!-- Dynamically change the width of your progressCompletion, ie 50% -->
-
-                </div>
-            </div>
-            <div class="funding">
-                <h4><strong>{{goal_funding}}</strong></h4> <!-- Input goal funding value -->
-                <p>{%trans "Goal" %}</p>
-            </div>
-        </div>
+  <a href="https://github.com/{{ github_username }}">
+    <img src="{{ github_username }}" />
+  </a>
+  <!-- width="50" height="50"Input project image here -->
+  <br>
+  <a class="grantInfoName" href="https://github.com/{{ github_username }}">{{ github_username }}</a>
+  <!-- Input project name here -->
+  <p>{{ grant_description }}</p>
+  <!-- Input project description here -->
+  <div class="progress">
+    <h3 style="text-transform: none;">{{ funding_status }}</h3>
```

Comment:
All of the template vars throughout this file need to reference the `Grant` object and it's associated attributes.